helmfile/charts/gatekeeper/podsecuritypolicies/templates/default/mutations/privilege-escalation.yaml

```diff
@@ -1,6 +1,15 @@
 {{- $namespaceSelectorLabels := .Values.mutations.namespaceSelectorLabels }}
 {{- $namespaces := keys .Values.constraints | sortAlpha }}
 {{- $exceptions := .Values.mutations.exceptions }}
+
+{{- range $namespace, $services := .Values.constraints }}
+{{- range $name, $value := $services }}
+{{- if $value | dig "allow" "allowPrivilegeEscalation" "true" }}
```

Comment:
It should be true right? As this mutation should not run if `allow.allowPrivilegeEscalation` is set to true. As we do not want it to add `false`

Reply:
No, since the `false` here is the default value if `allow.allowPrivilegeEscalation` is unset.
So having it as `true` will always skip the mutation unless it is explicitly set to `false`.

Reply:
Though come to think of it, the `"false"` should be `false` else it is will still evaluate to true.

Reply:
My understanding is the following:
1. If `allowPrivilegeEscalation` is set to `true`, add it as an exception so that it does not get mutated to false by this `assign`. (This was causing issues because the custom mutation would not set it to `true` as it was already set to `false` by this one from what I could tell. This was the reason for this change in the first place.) 
2. Otherwise, do not add this as an exception and let this `assign` set it to `false` regardless if you had it unset or explicitly set to `false`.

Please correct me if I'm completely missing something :smile:

Reply:
The problem here is that `$value | dig "allow" "allowPrivilegeEscalation" "true"` will evaluate to `"true"` if `.allow.allowPrivilegeEscalation` is unset. (The last arg passed to `dig`, not counting piped `$value`, is the default value it will evaluate to if it can't find a value.)
So that exception will always be added if `allowPrivilegeEscalation` isn't explicitly set to `false`, and that is not what you intend right?.

Reply:
Aaah now I see what you're saying, thanks for clarifying